From 76dbf3efc4cf512d3601f3431f04ca221e21397b Mon Sep 17 00:00:00 2001
From: Simon McVittie <smcv@debian.org>
Date: Mon, 7 Dec 2009 01:37:58 +0000
Subject: [PATCH] Revert a change to buildhtml.py that broke --local

The --local option relies on the semantics of os.path.walk, and can't
be adapted to os.walk quite this easily.

Forwarded: https://sourceforge.net/tracker/?func=detail&aid=2909811&group_id=38414&atid=422030
---
 tools/buildhtml.py |    9 ++-------
 1 files changed, 2 insertions(+), 7 deletions(-)

diff --git a/tools/buildhtml.py b/tools/buildhtml.py
index a5ac8b9..654800e 100755
--- a/tools/buildhtml.py
+++ b/tools/buildhtml.py
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-# $Id: buildhtml.py 5741 2008-12-01 07:21:03Z grubert $
+# $Id: buildhtml.py 5019 2007-03-12 21:48:30Z wiemann $
 # Author: David Goodger <goodger@python.org>
 # Copyright: This module has been placed in the public domain.
 
@@ -181,12 +181,7 @@ class Builder:
         else:
             self.directories = [os.getcwd()]
         for directory in self.directories:
-            try:
-                for root, dirs, files in os.walk(directory):
-                    self.visit(recurse, root, dirs+files)
-            except (AttributeError): # python2.2 does not have os.walk
-                print "no os.walk"
-                os.path.walk(directory, self.visit, recurse)
+            os.path.walk(directory, self.visit, recurse)
 
     def visit(self, recurse, directory, names):
         settings = self.get_settings('', directory)
-- 
1.6.5.4

